docs/docs/cloud/how-tos/remote_graph.md

```diff
@@ -0,0 +1,150 @@
+# How to interact with the deployment using RemoteGraph
+
+`RemoteGraph` is an interface that allows you to interact with your LangGraph Platform deployment as if it were a regular `CompiledGraph` instance. This guide shows you how you can initialize a `RemoteGraph` and interact with it.
+
+## Setup
```

Comment:
Set up? How to connect to a deployed server?

Reply:
yea need to rephrase the name